app/javascript/src/utilities/utils.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import {fetch as fetchPolyfill} from 'whatwg-fetch'
+import 'core-js/es6/promise'
+import React from 'react'
+
+function CSRFToken ({win = window}: {win?: Window}) {
+  const getMetaContent = meta => win.document.head.querySelector(`meta[name~=${meta}][content]`).content
+
+  try {
+    return (
+      <input
+        name={getMetaContent('csrf-param')}
+        value={getMetaContent('csrf-token')}
+        type='hidden'
+      />
+    )
+  } catch (error) {
+    console.error(error)
+    return null
```

Comment:
Here, as my previous comment, you could instead of returning `null`, could return the input with the value `null`. That way you'll be addressing the type signature implicit of the function. 
On a different matter, when @josemigallas suggested not to explicitly return the type of a `React.Component` I was fearing this case... since it would have failed the type check otherwise ;)